cmd/client-fs.go

```diff
@@ -246,12 +342,20 @@ func (f *fsClient) Put(reader io.Reader, size int64, contentType string, progres
 				return 0, probe.NewError(e)
 			}
 		}
+
 		// Allocate buffer of 10MiB once.
 		readAtBuffer := make([]byte, 10*1024*1024)
 
+		// Fetch the size of the reader.
+		var rsize int64
```

Comment:
Actually, size is already passed as a parameter in Put(), do we need to calculate it again?